var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -91,28 +108,33 @@ class Boost(Package):
         'chrono',
         'container',
         'context',
+        'contract',
         'coroutine',
         'date_time',
         'exception',
         'fiber',
         'filesystem',
         'graph',
+        'graph_parallel',
         'iostreams',
+        'json',
         'locale',
         'log',
         'math',
         'mpi',
+        'nowide',
         'program_options',
         'python',
         'random',
         'regex',
         'serialization',
-        'signals',
+        'stacktrace',
```

Comment:
```suggestion
        'signals',
        'stacktrace',
```

It shouldn't be there, but signals is still an allowed variant.